validator/validators/feature/feature_validator.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package feature
+
+import (
+	"time"
+
+	"go.uber.org/multierr"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/internal/awsservice"
+	"github.com/aws/amazon-cloudwatch-agent-test/internal/common"
+	"github.com/aws/amazon-cloudwatch-agent-test/validator/models"
+)
+
+type FeatureValidator struct {
+	vConfig models.ValidateConfig
+}
+
+var _ models.ValidatorFactory = (*FeatureValidator)(nil)
+
+func NewFeatureValidator(vConfig models.ValidateConfig) models.ValidatorFactory {
```

Comment:
Here is how the performance declare the validator through three stages at declared in the PR:

* [GenerateLoad](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/models/validator_factory.go#L10):
* [CheckData](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/models/validator_factory.go#L11)
* [Cleanup](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/models/validator_factory.go#L12)
However, as stated in the PR, we will only generate load in this PR with [metrics and logs only](https://github.com/aws/amazon-cloudwatch-agent-test/pull/151/files#diff-334efd747575193d3505843e769add29bb2818c73a18ca77c9c4f97105ec3104R35-R39) and clean up. The get and push validation will be in the upcoming PR.

Reply:
nit: the links seem to be off.